content/webapp/components/Body/Body.tsx

```diff
@@ -92,6 +93,17 @@ type Props = {
 
 type ContentListSlice = BodySlice & { type: 'contentList' };
 
+const Wrapper = styled(Space).attrs<{
+  rowBackgroundColor: string;
+  cardBackgroundColor: string;
+}>({
+  className: `row card-theme 
```

Comment:
we can lose the row class here too, since we've established it doesn't do anything